lib/Doctrine/DBAL/Schema/SQLServerSchemaManager.php

```diff
@@ -56,15 +56,15 @@ public function dropDatabase($database)
     /**
      * {@inheritdoc}
      */
-    protected function _getPortableSequenceDefinition($sequence)
+    protected function _getPortableSequenceDefinition($sequence) : Sequence
```

Comment:
The argument type doesn't match the parent definition. PhpStorm in the PHP 7.1 mode reports such violations but it doesn't in PHP 7.2 mode.